src/Dashboard/Web.config

```diff
@@ -1,100 +1,107 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0"?>
 <configuration>
   <connectionStrings>
-    <add name="AzureWebJobsDashboard" connectionString="" />
+    <add name="AzureWebJobsDashboard" connectionString=""/>
   </connectionStrings>
   <appSettings>
-    <add key="webpages:Version" value="3.0.0.0" />
-    <add key="webpages:Enabled" value="false" />
-    <add key="PreserveLoginUrl" value="true" />
-    <add key="ClientValidationEnabled" value="true" />
-    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-    
+    <add key="webpages:Version" value="3.0.0.0"/>
+    <add key="webpages:Enabled" value="false"/>
+    <add key="PreserveLoginUrl" value="true"/>
+    <add key="ClientValidationEnabled" value="true"/>
+    <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
     <!-- if specified, then the dashboard reads logs from this table instead of the 
     traditional blob logging -->
-    <add key="AzureWebJobsLogTableName" value="" />
+    <add key="AzureWebJobsLogTableName" value=""/>
   </appSettings>
+  <!--
+    For a description of web.config changes see http://go.microsoft.com/fwlink/?LinkId=235367.
+
+    The following attributes can be set on the <httpRuntime> tag.
+      <system.Web>
+        <httpRuntime targetFramework="4.6" />
+      </system.Web>
+  -->
   <system.web>
-    <customErrors mode="RemoteOnly" />
-    <compilation debug="true" targetFramework="4.5" />
-    <httpRuntime targetFramework="4.5" />
-    <authentication mode="None" />
+    <customErrors mode="RemoteOnly"/>
+    <compilation debug="true" targetFramework="4.6"/>
+    <httpRuntime targetFramework="4.5"/>
```

Comment:
It auto-added the runtime change above in a comment. I didn't want to do any manual adjusting here without a great way to test it out. My plan was just to leave it this way, figuring we could update it later.